wagtail/contrib/modeladmin/tests/test_simple_modeladmin.py

```diff
@@ -263,6 +263,43 @@ def test_col_extra_attributes(self):
         """
         self.assertContains(response, test_html, html=True)
 
+    def test_linked_name_column(self):
```

Comment:
```suggestion
    def test_title_column_links_to_edit_view_by_default(self):
```

Reply:
I will commit your suggestion with GH, but will properly change the tests locally (probably on monday).